### ISSUE TITLE
Make the toggle buttons more visible

### ISSUE THREAD

**Issue by vkbo** [vkbo/novelWriter#2827]
The toggle buttons, especially in the editor search box, are barely distinguishable between on and off when using a dark theme. Presumably due to how the default button shades are calculated.

These toggles should have a distinct colour instead.

**Comment by vkbo** [comment]
This could also be improved on light themes. The default Qt behaviour is to make a toggled on button 1.30 times darker. Regardless of how dark the button colour is.

What do you think, @HeyMyian? Should we add a "toggled" colour to the GUI theme? All themes would need an update. It would probably make sense to pick a version of the highlight or accent colour. Possibly not as bright as the default one though. I can give it a spin to see how it looks.

**Comment by vkbo** [comment]
I have added a new theme field named `toggle` and pre-populated with a variation of the accent colour. In most cases just a half-transparent copy. I'm probably going to tweak my themes, and if you want to modify yours, then just make a new PR on that.

It's easiest to inspect by opening the editor search and the project search, and toggle on a few buttons.

<img width="1369" height="359" alt="Image" src="https://github.com/user-attachments/assets/73094510-f829-4496-8ead-802a5868f7cc" />

**Comment by HeyMyian** [comment]
That sounds like a good change. Do these toggles occur anywhere else or only in the search sidebar and little search section in the corner, as is shown in your screenshot?

I had a look at the themes and yeah, the colors need to be tweaked in some themes. I will send you a PR for my themes within a couple of days.

**Comment by vkbo** [comment]
I couldn't find anywhere else that used these kind of toggle buttons. The other toggle buttons change icon instead to indicate on or off, so there the colour shouldn't change at all.

As for the colour, yeah, I found that especially on light themes, half transparency (128) was too much, so in many places I scaled down to 96 or even 64. For those using colour hex, 7f is half, but I put some as low as 57. I'll leave that to you for your themes. I've updated the others.

**Comment by HeyMyian** [comment]
Ah, good that you mentioned the transparency again. The transparency setting had no effect at all in my installation, probably because I had never updated PyQt?; I was still running PyQt 6.9.1 from last year when I installed novelWriter 2.7 with pip. I now installed novelWriter 26.1.1 with pip and pulled the current repository files into it - now the transparency works, with PyQt 6.11.1.

**Comment by vkbo** [comment]
Huh. I never noticed it not working. It also worked with Qt5 with the text highlight, so this is strange.

If you install uv, then run with `uv run novelwriter` it will always run with the correct package versions for development. They are pinned in the `uv.lock` file. This also ensures that the code formatting commands always do the correct thing if you were ever to change any code.

(I just updated the contributing guide since it was a bit out of date.)

**Comment by HeyMyian** [comment]
> Huh. I never noticed it not working. It also worked with Qt5 with the text highlight, so this is strange.

It was only the transparency on the toggle buttons; the other transparencies worked fine. I don't know exactly which of my outdated components caused it. I do think at this point it was solely an issue with my unorthodox method of installation and updating. Let me know however if you need more information.

> `uv run novelwriter`

Thanks, that seems like a much much smarter way of doing things than what I did :D
